src/PrestaShopBundle/Controller/Admin/Sell/CustomerService/MerchandiseReturnController.php

```diff
@@ -75,6 +83,95 @@ public function indexAction(Request $request, MerchandiseReturnFilters $filters)
         ]);
     }
 
+    /**
+     * Edit existing order return
+     *
+     * @AdminSecurity(
+     *     "is_granted(['update'], request.get('_legacy_controller'))",
+     *     redirectRoute="admin_merchandise_returns_index"
+     * )
+     *
+     * @param int $orderReturnId
+     * @param Request $request
+     *
+     * @return Response
+     */
+    public function editAction(int $orderReturnId, Request $request): Response
+    {
+        $formBuilder = $this->get('prestashop.core.form.identifiable_object.builder.order_return_form_builder');
+        $formHandler = $this->get('prestashop.core.form.identifiable_object.handler.order_return_form_handler');
+
+        try {
+            /** @var EditableOrderReturn $editableOrderReturn */
+            $editableOrderReturn = $this->getQueryBus()->handle(
+                new GetOrderReturnForEditing(
+                    $orderReturnId
+                )
+            );
+
+            $form = $formBuilder->getFormFor($orderReturnId);
+            $form->handleRequest($request);
+
+            $result = $formHandler->handleFor($orderReturnId, $form);
+
+            if ($result->getIdentifiableObjectId()) {
+                $this->addFlash('success', $this->trans('Update successful', 'Admin.Notifications.Success'));
+
+                return $this->redirectToRoute('admin_merchandise_returns_index');
+            }
+        } catch (Exception $e) {
+            $this->addFlash('error', $this->getErrorMessageForException($e, $this->getErrorMessages()));
+
+            return $this->redirectToRoute('admin_merchandise_returns_index');
+        }
+
+        $legacyContext = $this->get('prestashop.adapter.legacy.context');
+
+        return $this->render('@PrestaShop/Admin/Sell/CustomerService/OrderReturn/edit.html.twig', [
+            'help_link' => $this->generateSidebarLink($request->attributes->get('_legacy_controller')),
+            'enableSidebar' => true,
+            'layoutTitle' => sprintf($this->trans('Return Merchandise Authorization (RMA)', 'Admin.Orderscustomers.Feature')),
```

Comment:
Why do you need a `sprintf` here? You have no argument to inject, and even if you did `trans` can handle that

Reply:
No idea why it's there. Nice catch!